pkg/api/postage.go

```diff
@@ -37,6 +37,26 @@ func (s *Service) postageAccessHandler(h http.Handler) http.Handler {
 	})
 }
 
+func (s *Service) postageSyncStatusCheckHandler(h http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		done, err := s.syncStatus()
+		if err != nil {
+			s.logger.Debugf("postage access: %v", err)
+			s.logger.Error("postage access: syncing failed")
+			jsonhttp.ServiceUnavailable(w, "batch unavailable: syncing failed")
```

Comment:
the error message here is a bit nebulous, also, not sure why "batch unavailable".

Reply:
fixed